clients/drcachesim/tracer/output.cpp

```diff
@@ -1163,6 +1210,7 @@ init_thread_io(void *drcontext)
     per_thread_t *data = (per_thread_t *)drmgr_get_tls_field(drcontext, tls_idx);
     byte *proc_info;
 
+    NOTIFY(1, "T%d: in init_thread_io.\n", dr_get_thread_id(drcontext));
```

Comment:
nit: This is actually pointer-sized on Windows and 64-bit on Mac. TIDFMT can be used.

Reply:
Fixed.